packages/components/src/text-control/index.tsx

```diff
@@ -46,7 +48,13 @@ function UnforwardedTextControl(
 			className={ className }
 		>
 			<input
-				className="components-text-control__input"
+				className={ classnames(
+					className,
```

Comment:
It looks like `classname` is already passed to `BaseControl`, so probably we shouldn't pass it to the `input` element too?